Vagrantfile

```diff
@@ -2,7 +2,7 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "hashicorp/precise64"
+  config.vm.box = "bento/ubuntu-16.04"
```

Comment:
the bento boxes are generally fairly good and of known origin (the packer manifests are open).